sdk/typescript/src/client/types/generated.ts

```diff
@@ -713,6 +713,8 @@ export interface IotaSystemStateSummary {
     safeMode: boolean;
     /** Amount of computation rewards accumulated (and not yet distributed) during safe mode. */
     safeModeComputationRewards: string;
+    /** Amount of burned computation rewards accumulated during safe mode. */
+    safeModeComputationRewardsBurned: string;
```

Comment:
```suggestion
    safeModeComputationCharges: string;
    /** Amount of burned computation charges accumulated during safe mode. */
    safeModeComputationChargesBurned: string;
```

Reply:
This suggestion will be outdated due to https://github.com/iotaledger/iota/pull/5270/commits/3d65d4a39724a007ab66be516b08e4bb0fc44c9d and https://github.com/iotaledger/iota/pull/5270/commits/3f5933339b60e7b381332c09e578516965fc801c

`IotaSystemStateSummaryV1` will now be kept unchanged with field `safe_mode_computation_rewards` and `IotaSystemStateSummaryV2` introduced with renamed and additional fields.